syntax = "proto3";

package motify.music;

service SearchService {
  rpc Find(SearchRequest) returns (SearchResponse) {}
}

message SearchRequest{
  string keyword = 1;
}

message SearchResponse{
  repeated Song songs = 1;
}

message Song{
  string id = 1;
  string name = 2;
  string artist = 3;
  string album = 4;
  string genre = 5;
  string date_added = 6;
}